rotkehlchen/tests/unit/decoders/test_curve_lend.py

```diff
@@ -0,0 +1,694 @@
+            balance=Balance(amount=FVal(return_amount)),
+            location_label=user_address,
+            notes=f'Return {return_amount} cvcrvUSD to a Curve vault',
+            counterparty=CPT_CURVE,
+            address=ZERO_ADDRESS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.OPTIMISM,
+            event_type=HistoryEventType.WITHDRAWAL,
+            event_subtype=HistoryEventSubType.REMOVE_ASSET,
+            asset=underlying_token,
+            balance=Balance(amount=FVal(withdraw_amount)),
+            location_label=user_address,
+            notes=f'Withdraw {withdraw_amount} crvUSD from a Curve vault',
+            counterparty=CPT_CURVE,
+            address=string_to_evm_address('0x9dba46e6a06FBf24CA11f8912B44338fe1b28Ea9'),
+        ),
+    ]
+
+
+@pytest.mark.vcr(filter_query_parameters=['apikey'])
+@pytest.mark.parametrize('ethereum_accounts', [['0x1D6702bD3DA0108a4428415FFc74B0efd2F86E4f']])
+def test_create_loan(
+        ethereum_inquirer: 'EthereumInquirer',
+        ethereum_accounts: list['ChecksumEvmAddress'],
+) -> None:
+    tx_hash = deserialize_evm_tx_hash('0xcaaba5f6206de1a1d3a82a213616548b928397052300057a420ff8083d385b78')  # noqa: E501
+    _, underlying_token = _create_ethereum_vault_tokens(database=ethereum_inquirer.database)
+    events, _ = get_decoded_events_of_transaction(evm_inquirer=ethereum_inquirer, tx_hash=tx_hash)
+    timestamp, user_address, gas_amount, deposit_amount, receive_amount = TimestampMS(1731831407000), ethereum_accounts[0], '0.004991520607076806', '0.00011219', '6.178023671273738089'  # noqa: E501
+    assert events == [
+        EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=0,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.SPEND,
+            event_subtype=HistoryEventSubType.FEE,
+            asset=A_ETH,
+            balance=Balance(amount=FVal(gas_amount)),
+            location_label=user_address,
+            notes=f'Burn {gas_amount} ETH for gas',
+            counterparty=CPT_GAS,
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=1,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.DEPOSIT,
+            event_subtype=HistoryEventSubType.DEPOSIT_ASSET,
+            asset=A_WBTC,
+            balance=Balance(amount=FVal(deposit_amount)),
+            location_label=user_address,
+            notes=f'Deposit {deposit_amount} WBTC as collateral on Curve',
+            counterparty=CPT_CURVE,
+            address=string_to_evm_address('0x8eeDE294459EFaFf55d580bc95C98306Ab03F0C8'),
+        ), EvmEvent(
+            tx_hash=tx_hash,
+            sequence_index=2,
+            timestamp=timestamp,
+            location=Location.ETHEREUM,
+            event_type=HistoryEventType.RECEIVE,
```

Comment:
Can you please change those to WITHDRAWAL/GENERATE_DEBT. These are the types we use for similar protocols such as liquity